net/zerotier/Makefile

```diff
@@ -0,0 +1,88 @@
+#
```

Comment:
Don't set copyright to OpenWrt unless you're affiliated or under contract.
Set yourself as copyright holder or omit it.